src/traces/isosurface/attributes.js

```diff
@@ -225,9 +225,13 @@ var attrs = module.exports = overrideAll(extendFlat({
             'If trace `hoverinfo` contains a *text* flag and *hovertext* is not set,',
             'these elements will be seen in the hover labels.'
         ].join(' ')
-    },
+    }
 },
 
+opacityscaleAttrs('', {
```

Comment:
Hmm. Does this actual do anything? Do we really want to add `opacityscale` to `isosurface` traces?